libs/aws/langchain_aws/chat_models/bedrock_converse.py

```diff
@@ -659,7 +659,7 @@ def _anthropic_to_bedrock(
         elif block["type"] == "image":
             # Assume block is already in bedrock format.
             if "image" in block:
-                bedrock_content.append(block)
+                bedrock_content.append({"image": block["image"]})
```

Comment:
could we add/update unit test for this?

Reply:
Thanks for calling this out. Looks like, both `image` and `document` are not supported in the output at the moment, so we should remove both.
https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_Message.html

Let me know if I am missing anything.

Reply:
Added unit tests.